composer.json

```
@@ -12,13 +12,14 @@
         }
     },
     "require": {
-        "php": "^7.1",
+        "php": "^8.1",
```

If you still need 7.x support, I think you can do something like this
```suggestion
        "php": "^7.1|^8.1",
```

---

We certainly hope not to need PHP 7.x support 🤣

---

Yeah, I think if anybody needs 7.X support they can just use an old release at this point.